Alexey/Mortgage Calculator.py

```diff
@@ -0,0 +1,42 @@
+answer = input("Choose type of calc y/m/w/d/: ")
+P = int(input("Principal Amount (initial loan balance): "))
+i = float(input("Interest Rate: ")) / 100
+t = int(input("Count of Years: "))
+n = t * 12

```

Comment:
This should be moved to main method or to the `if __name__ == "__main__":` statement